.taskfiles/install.yml

```diff
@@ -22,7 +22,7 @@
 version: "3"
 
 vars:
-  GOLANGCI_LINT_VERSION: v1.61.0
+  GOLANGCI_LINT_VERSION: v1.62.2
```

Comment:
Can you also update the golangci-lint version in `.github/workflows/golangci-lint.yml`? Not the action version but the golangci-lint version. I think it should be

```yaml
      - name: Run golangci-lint
        uses: golangci/golangci-lint-action@971e284b6050e8a5849b72094c50ab08da042db8 # v6.1.1
        with:
          install-mode: goinstall
          version: 89476e7a1eaa0a8a06c17343af960a5fd9e7edb7 # v1.62.2
```

Reply:
Thanks!